frontend/src/static/js/pages/components/MediaListHeader.js

```
@@ -1,12 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
 export function MediaListHeader(props){
 	return (<><div className={ ( void 0 === props.className || null === props.className ? '' : props.className + ' ' ) + 'hw-category' } style={ props.style }>
 				<h2 className="hw-category-title">{ props.title }</h2>
 				{ void 0 === props.viewAllLink || null === props.viewAllLink ? null : <a href={ props.viewAllLink } title={ props.viewAllText || props.viewAllLink } className="hw-category-link">{ props.viewAllText || props.viewAllLink }</a> }
 			</div>
-			{ props.desc && <div className="hw-category-description">{props.desc}</div> }
+			{ props.desc && <div className="hw-category-description" dangerouslySetInnerHTML={{ __html: props.desc }}></div> }
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)